### ISSUE TITLE
New-NavContainer error "Container BC16RTM does not exist"

### ISSUE THREAD

**Issue by ti-jalopez** [microsoft/navcontainerhelper#1153]
**Describe the issue**
I have two containers created: BC16RTM and BC16RTM2 using New-NavContainer.
I have deleted container BC16RTM using Remove-NavContainer and docker container rm BC16RTM
I try to recreate BC16RTM using New-NavContainer, but it throws error "Container BC16RTM does not exist".
I debug NavContainerHelper powershell sctipts, a detect de error is in function Test-NavContainer that throws the error because docker ps -filter=BC16RTM get a container BC16RTM2.
To solve this I test to use "docker ps -filter='BC16RTM'   (container name is quoted) that is ok

**Scripts used to create container and cause the issue**
```
$containerName = 'BC16RTM'
$myLicense = 'xxxxxf'
$navcredential = New-Object System.Management.Automation.PSCredential -argumentList "nav", (ConvertTo-SecureString -String "!2020NavTest" -AsPlainText -Force)
$myArtifactUrl = Get-BCArtifactUrl -type OnPrem -version 16.0 -country es #Ejemplo  version especifica BC 16 RTM (16.0)
New-NavContainer -auth NavUserPassword -Credential $navcredential -updateHosts -accept_eula -containername $containerName -artifactUrl $myArtifactUrl -licenseFile $myLicense -accept_outdated -includeAL -includeTestToolkit

$containerName = 'BC16RTM2'
New-NavContainer -auth NavUserPassword -Credential $navcredential -updateHosts -accept_eula -containername $containerName -artifactUrl $myArtifactUrl -licenseFile $myLicense -accept_outdated -includeAL -includeTestToolkit

#when recreate BC16RTM thows the error
$containerName = 'BC16RTM'
Remove-NavContainer $containerName 
New-NavContainer -auth NavUserPassword -Credential $navcredential -updateHosts -accept_eula -containername $containerName -artifactUrl $myArtifactUrl -licenseFile $myLicense -accept_outdated -includeAL -includeTestToolkit

```

**Full output of scripts**
```
PS C:\WINDOWS\system32>
$containerName = 'BC16RTM'
New-NavContainer -auth NavUserPassword -Credential $navcredential -updateHosts -accept_eula -containername $containerName -artifactUrl $myArtifactUrl -licenseFile $myLicense -accept_outdated -includeAL -includeTestToolkit

ArtifactURL=https://bcartifacts.azureedge.net/onprem/16.0.11240.12085/es
NavContainerHelper is version 0.7.0.18
NavContainerHelper is running as administrator
Host is Microsoft Windows 10 Pro - 2004
Docker Client Version is 19.03.8
Docker Server Version is 19.03.8
Container BC16RTM does not exist
En C:\Program Files\WindowsPowerShell\Modules\navcontainerhelper\0.7.0.18\ContainerInfo\Get-NavContainerId.ps1: 34 Carácter: 13
+             throw "Container $containerName does not exist"
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Container BC16RTM does not exist:String) [], RuntimeException
    + FullyQualifiedErrorId : Container BC16RTM does not exist
...
```

**Screenshots**

**Additional context**
Test-NavContainer.ps1: error in --filter param without quotes in container name
==============
function Test-NavContainer {
    Param (
        [Parameter(Mandatory=$true)]
        [string] $containerName,

        [switch] $doNotIncludeStoppedContainers
    )
    Process {
        $id = ""
        $a = "-a"
        if ($doNotIncludeStoppedContainers) {
            $a = ""
        }
        $id = docker ps $a -q --no-trunc --filter "name=$containerName"
        if (!($id)) {
            $id = docker ps $a -q --no-trunc --filter "id=$containerName"
        }
        if ($id) {
            $inspect = docker inspect $id | ConvertFrom-Json
            ($inspect.Config.Labels.psobject.Properties.Match('maintainer').Count -ne 0 -and $inspect.Config.Labels.maintainer -eq "Dynamics SMB")
        } else {
            $false
        }
    }
}

**Comment by freddydk** [comment]
The problem is not quoting, but the fact that

docker ps --filter "name=bc"

will give you all containers with the bc name in it - and you cannot ask for a specific name.
I will change the Test function to not filter on name.

**Comment by freddydk** [comment]
The lines in Test-NavContainer should be changed to:

```
        $id = docker ps $a -q --no-trunc --format "{{.ID}}/{{.Names}}" | Where-Object { $containerName -eq $_.split('/')[1] } | % { $_.split('/')[0] }
        if (!($id)) {
            $id = docker ps $a -q --no-trunc --filter "id=$containerName"
        }
```

**Comment by freddydk** [comment]
Shipped in 0.7.0.19

**Comment by ti-jalopez** [comment]
I've tested the fix and run fine. Thank you :=)
